src/fides/api/models/privacy_preference.py

```diff
@@ -44,16 +45,12 @@ class ConsentMethod(Enum):
     individual_notice = "individual_notice"
 
 
-class PrivacyPreferenceHistory(Base):
-    """The DB ORM model for storing PrivacyPreferenceHistory, used for saving
-    every time consent preferences are saved for reporting purposes.
+class ConsentReportingMixin:
```

Comment:
This `ConsentReportingMixin` extracts existing fields from the`PrivacyPreferenceHistory` to share with the `ServedNoticeHistory` tables as we want to track most of the same information.  We are storing them separately because there are a few fields that are unique to each, and one record can exist without the other and vice versa.